Mathlib/Analysis/SpecialFunctions/Complex/Arg.lean

```diff
@@ -491,6 +491,20 @@ theorem arg_div_coe_angle {x y : ℂ} (hx : x ≠ 0) (hy : y ≠ 0) :
     (arg (x / y) : Real.Angle) = arg x - arg y := by
   rw [div_eq_mul_inv, arg_mul_coe_angle hx (inv_ne_zero hy), arg_inv_coe_angle, sub_eq_add_neg]
 
+theorem arg_pow_coe_angle {x : ℂ} {n : ℕ} :
+    ((x ^ n).arg : Real.Angle) = n • (x.arg : Real.Angle) := by
+  obtain rfl | x0 := eq_or_ne x 0
+  · by_cases n0 : n = 0 <;> simp [n0]
+  · induction n with
+    | zero => simp [x0]
+    | succ n ih => rw [pow_succ, arg_mul_coe_angle (pow_ne_zero n x0) x0, ih]; rfl
```

Comment:
If you replace this `rfl` with `simp?`, does it tell you a lemma name that you can add to the rewrite?

Reply:
`simp?` just give me an error saying "simp made no progress"`
I also tried `apply?`, but it gave me `exact rfl`, which is why I put `rfl` there

Reply:
`succ_nsmul` is the name of the lemma, but I had to guess it
```suggestion
    | succ n ih => rw [pow_succ, arg_mul_coe_angle (pow_ne_zero n x0) x0, ih, succ_nsmul]
```